README.md

```diff
@@ -26,41 +28,45 @@ composer require jolicode/elastically
 
 ## Demo
 
+> If you are using Symfony, you can move to the Symfony [chapter](#usage-in-symfony)
+
 Quick example of what the library do on top of Elastica:
 
 ```php
 // Your own DTO, or one generated by Jane (see below)
 class Beer
 {
-    public $foo;
-    public $bar;
+    public string $foo;
+    public string $bar;
 }
 
-// Building the Index from a mapping config
-use JoliCode\Elastically\Client;
+use JoliCode\Elastically\Factory;
 use Elastica\Document;
 
-// New Client object with new options
-$client = new Client([
+// New Factory object with new options
```

Comment:
The logic is way different here. Before it was just an Elastica client compatible client.
Now we have to build a Factory, not a Client.

```suggestion
// Factory object with Elastica options + new Elastically options in the same array
```